blueskyacademy/bluesky-nextjs
# add form into parent page

## Description
- Add form to Parent Page
- Add video to HomePage
- Add Faq to HomePage


## Why
- Users can register visit school tour

## How to QA
- Users can register visit school tour
- User can see a video introduction about BSA on HomePage
- User can see a list of frequently asking questions on HomePage

## Screenshots
![Screen Shot 2021-12-27 at 07 28 03](https://user-images.githubusercontent.com/74655110/147423588-1034004b-f7ab-44a5-a8a7-0aa9554a8785.png)
![Screen Shot 2021-12-27 at 07 27 49](https://user-images.githubusercontent.com/74655110/147423590-ac616ba8-ee22-4fc5-be98-7b1069d7ecfa.png)
